bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/PersistenceService.java

```diff
@@ -70,4 +72,11 @@
      * @param alias the alias under which the item should be persisted.
      */
     void store(Item item, String alias);
+
+    /**
+     * Provides default persistence strategies that are used for all items if no user defined configuration is found.
+     *
+     * @return The default persistence strategies
+     */
+    List<PersistenceStrategy> getDefaultStrategies();
```

Comment:
I know that this PR is APIBreaking anyways but you also can add the default method implementation here:
```suggestion
    default List<PersistenceStrategy> getDefaultStrategies() {
        return Collections.emptyList();
    }
```
Be aware of an eventually missing import.

Reply:
I thought about this, but I decided against it. Problem with the default methods is that no method skeleton is created in implementing classes, so that developers are not aware that they should actually think about implementing the method.